<ui version="4.0" >
 <class>SearchDialogClass</class>
 <widget class="QDialog" name="SearchDialogClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>282</height>
   </rect>
  </property>
  <property name="maximumSize" >
   <size>
    <width>400</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Search dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4" >
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="maximumSize" >
      <size>
       <width>400</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="currentIndex" >
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab" >
      <property name="geometry" >
       <rect>
        <x>4</x>
        <y>34</y>
        <width>330</width>
        <height>237</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Reactions</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2" >
       <item rowspan="2" row="0" column="0" colspan="2" >
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Left metabolites</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2" >
        <widget class="MyTextEdit" name="textEditLeftMetabolites" >
         <property name="minimumSize" >
          <size>
           <width>110</width>
           <height>0</height>
          </size>
         </property>
         <property name="tabChangesFocus" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="3" >
        <widget class="MyTextEdit" name="textEditRightMetabolites" >
         <property name="minimumSize" >
          <size>
           <width>110</width>
           <height>0</height>
          </size>
         </property>
         <property name="tabChangesFocus" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2" >
        <widget class="QLabel" name="label_3" >
         <property name="text" >
          <string>Direction of reaction</string>
         </property>
        </widget>
       </item>
       <item rowspan="2" row="4" column="0" colspan="2" >
        <widget class="QLabel" name="label_7" >
         <property name="text" >
          <string>Organism</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1" colspan="3" >
        <widget class="QComboBox" name="comboBox_4" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize" >
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="currentIndex" >
          <number>-1</number>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="3" >
        <spacer name="horizontalSpacer_3" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>182</width>
           <height>22</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="3" >
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>Right metabolites</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2" >
        <widget class="QComboBox" name="comboBoxDirection" >
         <property name="maximumSize" >
          <size>
           <width>90</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="6" column="3" >
        <widget class="QPushButton" name="pushButton" >
         <property name="text" >
          <string>Search</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>330</width>
        <height>237</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Compounds</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout" >
       <item row="0" column="0" >
        <widget class="QGroupBox" name="groupBoxName" >
         <property name="maximumSize" >
          <size>
           <width>16777215</width>
           <height>60</height>
          </size>
         </property>
         <property name="title" >
          <string>Name/Id (ie: citrate or C00158)</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3" >
          <item>
           <widget class="QLineEdit" name="lineEditNameID" />
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QGroupBox" name="groupBoxFormula" >
         <property name="maximumSize" >
          <size>
           <width>16777215</width>
           <height>60</height>
          </size>
         </property>
         <property name="title" >
          <string>Formula (ie: C6 )</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2" >
          <item>
           <widget class="QLineEdit" name="lineEditFormula" />
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0" >
        <spacer name="horizontalSpacer" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>197</width>
           <height>22</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1" colspan="3" >
        <widget class="QPushButton" name="pushButton_2" >
         <property name="text" >
          <string>Search</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QGroupBox" name="groupBox_3" >
         <property name="maximumSize" >
          <size>
           <width>16777215</width>
           <height>80</height>
          </size>
         </property>
         <property name="title" >
          <string>Search criterion</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout" >
          <item>
           <widget class="QRadioButton" name="radioButtonNameID" >
            <property name="text" >
             <string>Name/Id</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButtonFormula" >
            <property name="text" >
             <string>Formula</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>MyTextEdit</class>
   <extends>QTextEdit</extends>
   <header>mytextedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
